README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Short Biography
+
+![](images/IMG_9165.png)
+
+**Sandra Mauro**
+
+My name is Sandra Mauro and I am a first-year student at UChicago.
+I am currently in the SISRM program.
+
+
+These are my summer plans:
+ + Take Computing for the Social Sciences
+ + Conduct research with Professor Casey Mulligan
+ + Take an online python course
```

Comment:
Learning both R and Python at the same time could be helpful! I think it helped me understand both languages better and faster. But things can get confusing sometimes too 😆